test/tests/functional/cgroups_tests.py

```diff
@@ -52,7 +52,7 @@
                     "eatmem.c"]
 
 
-class CgroupsTests(PBSTestSuite):
+class TestCgroups(TestFunctional):

```

Comment:
Looks like we have to rename this file as per test file naming conventions. i.e. pbs_featurename.py